CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.1)
-project(modbuspp)
+project(modbuspp VERSION 1.0.0)
```

Comment:
Usually version # > 1.0 means it is safe to enter production. I would following the release version to be 0.3